src/components/Common/OpenPositions.js

```diff
@@ -41,26 +39,20 @@ const renderJobsForlocation = (jobs, location, key) => {
   )
 }
 
-const OpenPositions = React.forwardRef(
-  ({ data: { title, getInTouchText, getInTouchTitle } }, ref) => (
-    <div ref={ref}>
-      <Section greyBg id="open-positions">
-        <Padding top={{ smallPhone: 3, tablet: 4 }}>
-          <SectionTitle>{title}</SectionTitle>
-          <JobsByLocation>
-            {jobs =>
-              jobs.map(({ location, jobs: jobsForLocation }, idx) =>
-                renderJobsForlocation(jobsForLocation, location, idx)
-              )
-            }
-          </JobsByLocation>
-          <Padding top={{ smallPhone: 3, tablet: 4 }}>
-            <Hr />
-          </Padding>
-        </Padding>
-      </Section>
-      <GetInTouch title={getInTouchTitle} contactText={getInTouchText} />
-    </div>
-  )
-)
+const OpenPositions = React.forwardRef(({ data: { title } }, ref = null) => (
```

Comment:
We're missing a point from the ticket here

```Create limit prop to display a limited number of jobs on homepage. See @jameshevey3 comment below```

Currently it's set up so that if Nick were to add 20 jobs to lever, they would all display on the homepage which isn't what we want. 

- Add a limit prop to here then pass it to `JobsByLocation`, I would say we can use this variable as a limit filter in the graphql query but unfortunately we need to limit on the location not all lever jobs as a whole.

Reply:
```I would say we can use this variable as a limit filter in the graphql query but unfortunately we need to limit on the location not all lever jobs as a whole.```

^ ignore this, I looked a bit closer at the current query in place and it turns out we can do this.

Check it out [here](http://localhost:8000/___graphql?query=query%20JOBS_BY_CITY(%24groupLimit%3A%20Int)%20%7B%0A%20%20allLever(limit%3A%2040)%20%7B%0A%20%20%20%20group(field%3A%20categories___location%2C%20limit%3A%20%24groupLimit)%20%7B%0A%20%20%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20location%0A%20%20%20%20%20%20%20%20%20%20%20%20commitment%0A%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%20%20text%0A%20%20%20%20%20%20%20%20%20%20hostedUrl%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&operationName=JOBS_BY_CITY&variables=%7B%0A%20%20%22groupLimit%22%3A%204%0A%7D)

Reply:
ahah im going to back track again, we can't do it this way because we can't pass variables to the StaticQuery from above,  this is a design limitation with Gatsby unfortunately. We will have to transform the data coming back from the query